src/client.ts

```diff
@@ -399,16 +399,18 @@ export default class Client implements ClientInterface {
 
   private async projectHasDependency(gemName: string): Promise<boolean> {
     try {
+      // We can't include `BUNDLE_GEMFILE` here, because we want to check if the project's bundle includes the
+      // dependency and not our custom bundle
+      const withoutBundleGemfileEnv = { ...this.ruby.env };
```

Comment:
The idiomatic way to do this is by using [Rest in Object Destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Rest_in_Object_Destructuring) operation:
```suggestion
      const { BUNDLE_GEMFILE, ...withoutBundleGemfileEnv } = this.ruby.env;
```

Reply:
Very nice.